README.md

```diff
@@ -5,73 +5,41 @@ Licensed under the Apache License, Version 2.0 (the LICENSE-APACHE file) or
 the MIT license (the LICENSE-MIT file) at your option. This file may not be
 copied, modified, or distributed except according to those terms. -->
 
-# Allocators, general and specific
-
-This repo includes a number of custom dynamic memory allocators written in Rust,
-along with some utilities used to implement these allocators. The `info`
-directory contains more detailed information about specific components of the
-project (e.g. performance measurements). Aside from `info`, all top-level
-directories are Rust crates. More detailed information on each crate can be
-found in each top-level directory's readme.
-
-## Allocator Crates
-
-* `slab-alloc` includes an implementation of a slab allocator in the same
-  tradition as the original [slab allocator](https://www.usenix.org/legacy/publications/library/proceedings/bos94/full_papers/bonwick.a)
-  by Jeff Bonwick (though several details are changed from the original). Some
-  key points to consider here are:
-
-  * Slab allocators are object-specific: they can only create objects of a
```

Comment:
I think this content is lost in this change (i.e. people can't read about these by just looking at the slab-alloc docs; maybe move some of it there?)